docs/usage/cli.md

```diff
@@ -0,0 +1,35 @@
+# Command Line Interface (CLI)
+
+## Labelmerge CLI
+The following can also be seen by entering `labelmerge -h` into your terminal.
+
+These are all the required and optional arguments Labelmerge accepts in order to 
+run flexibly on many different input data types and with many options. In most 
```

Comment:
I don't even really think the last bit is true -- you're almost always going to need either `--overlay_bids dir` or `--base_desc` and `--overlay_desc`

Reply:
changed to include thse